webrtc/pending-promise-after-close.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>Test pending promises behavior when RTCPeerConnection object is closed</title>
+</head>
+<body>
+  <!-- These files are in place when executing on W3C. -->
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script type="text/javascript">
+    'use strict'
+
+    function test_never_resolved(test_func, testName) {
+      async_test(t => {
+        const pc = new RTCPeerConnection();
+
+        test_func(pc).then(
+          t.step_func(result => {
+            assert_unreached(`Pending promise should never be resolved. Instead it is fulfilled with: ${result}`)
+          }),
+          t.step_func(err => {
+           assert_unreached(`Pending promise should never be resolved. Instead it is rejected with: ${err}`)
+          }));
+
+        // Close the PeerConnection before enqueued operation is executed.
+        pc.close();
+
+        t.step_timeout(() => {
+          t.done();
+        }, 2000);
```

Comment:
One way to avoid timeouts in a test like this is to use another chain of events that we know would be completed after the first. So, you could create a second RTCPeerConnection where you run the same steps and then don't call `close()`. Once that is done, if the first hasn't been resolved or rejected yet, then the test can end. Probably the test would run faster, although use more resources. Whatever you think would be nicer.